bin/client.py

```diff
@@ -292,10 +292,16 @@ def main():
                           default='/etc/apel/sender.cfg')
 
     opt_parser.add_option('-l', '--log_config',
-                          help='location of logging config file (optional)',
-                          default='/etc/apel/logging.cfg')
+                          help='DEPRECATED - location of logging config file (optional)',
+                          default=None)
 
     options, unused_args = opt_parser.parse_args()
+
+    # Deprecating functionality.
+    old_log_config_default_path = '/etc/apel/logging.cfg'
+    if (os.path.exists(old_log_config_default_path) or options.log_config is not None):
+        logging.warning('Separate logging config file option has been deprecated.')
```

Comment:
It would be good to move this after line 316, after the local logger is set up so that we can do `log.warning` so that the logging goes to the actual location that's configured. (Same for each file.)

The outer parentheses on line 302 are unnecessary.